script/build

```diff
@@ -52,18 +53,21 @@ process.on('unhandledRejection', function (e) {
   process.exit(1)
 })
 
-checkChromedriverVersion()
-cleanOutputDirectory()
-copyAssets()
-transpilePackagesWithCustomTranspilerPaths()
-transpileBabelPaths()
-transpileCoffeeScriptPaths()
-transpileCsonPaths()
-transpilePegJsPaths()
-generateModuleCache()
-prebuildLessCache()
-generateMetadata()
-generateAPIDocs()
+if (!argv.existingArtifacts) {
+  checkChromedriverVersion()
+  cleanOutputDirectory()
+  copyAssets()
+  transpilePackagesWithCustomTranspilerPaths()
+  transpileBabelPaths()
+  transpileCoffeeScriptPaths()
+  transpileCsonPaths()
+  transpilePegJsPaths()
+  generateModuleCache()
+  prebuildLessCache()
+  generateMetadata()
+  generateAPIDocs()
+}
+
 dumpSymbols()
```

Comment:
Could `dumpSymbols` be moved inside the `existingArtifacts` conditional as well? Maybe something like

```js
let artifactsPromise = Promise.resolve()

if (!argv.existingArtifacts) {
  checkChromedriverVersion()
  cleanOutputDirectory()
  copyAssets()
  transpilePackagesWithCustomTranspilerPaths()
  transpileBabelPaths()
  transpileCoffeeScriptPaths()
  transpileCsonPaths()
  transpilePegJsPaths()
  generateModuleCache()
  prebuildLessCache()
  generateMetadata()
  generateAPIDocs()
  artifactsPromise = dumpSymbols()
}

artifactsPromise()
  .then(packageApplication)
  .then // ...
```